consensus/src/lib.rs

```diff
@@ -7,7 +7,7 @@
 //! Consensus for the Aptos Core blockchain
 //!
 //! The consensus protocol implemented is AptosBFT (based on
-//! [HotStuff](https://arxiv.org/pdf/1803.05069.pdf)).
+//! [DiemBFT](https://developers.diem.com/papers/diem-consensus-state-machine-replication-in-the-diem-blockchain/2021-08-17.pdf)).
```

Comment:
refer to Joleton instead?

Reply:
That PDF is more accurate to what we implement in case someone will actually read.